source/PlayerInfo.cpp

```diff
@@ -107,8 +104,10 @@ void PlayerInfo::New(const StartConditions &start)
 	SetSystem(start.GetSystem());
 	SetPlanet(&start.GetPlanet());
 	accounts = start.GetAccounts();
-	start.GetConditions().Apply(conditions);
+	// Register the derived conditions before applying start conditions, so that any that belong in a derived provider go into that correctly.
 	RegisterDerivedConditions();
+	start.GetConditions().Apply(conditions);
+	// RegisterDerivedConditions() used to be here
```

Comment:
This fix looks good. Please just remove the out-commented code to keep the code-base clean. (Also for out-commented code later.)

Reply:
Removed. My comments are still a bit verbose: wasn't sure where's the best balance.